pkgs/development/tools/build-managers/gradle/default.nix

```diff
@@ -99,6 +108,22 @@ rec {
         echo ${ncurses6} >> $out/nix-support/manual-runtime-dependencies
       '';
 
+      passthru.tests = {
```

Comment:
fails on darwin

```
FAILURE: Build failed with an exception.
* What went wrong:
Gradle could not start your build.
> Could not initialize native services.
   > Failed to load native library 'libnative-platform.dylib' for Mac OS X x86_64.
* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
gradle --version returned a non-zero exit code.
FAILURE: Build failed with an exception.
* What went wrong:
Gradle could not start your build.
> Could not initialize native services.
   > Failed to load native library 'libnative-platform.dylib' for Mac OS X x86_64.
* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
error: builder for '/nix/store/pdfs37wfivmii12mqmx95xp6vlzn53qz-actual.drv' failed with exit code 1;
```

Reply:
That seems to suggest that Gradle doesn’t work on macOS at all. Guess I’ll have to try to find a macOS to test with.

Reply:
Turns out Gradle wants to extract the library to `$HOME/native`, which is not writable on macOS, apparently. Will try to figure out how to work around that.

Reply:
Should work on macOS now.